ts/components/ui/BannerErrorState.tsx

```diff
@@ -97,7 +100,9 @@ export const BannerErrorState = ({
         )}
         {label && (
           <>
-            <Body color={colorContent}>{label}</Body>
+            <Body color={colorContent} textStyle={styles.textCenter}>
```

Comment:
There's no need to define a new style object if we use it once. We can just insert it because the performance improvements are truly marginal.
```suggestion
            <Body color={colorContent} textStyle={{ textAlign: "center" }}>
```

Reply:
Is adding an inline style necessary? I created a style object not for performance reasons but simply for better readability.

Reply:
I think that readability is the same, inline style or not